docs/source/Tutorial.md

```diff
@@ -2472,10 +2472,10 @@ myGame.Monster monster = new myGame.Monster(data);
 <div class="language-swift">
 ~~~{.swift}
   // create a ByteBuffer(:) from an [UInt8] or Data()
-  let buf = // Get your data
-
+  var buf = // Get your data
   // Get an accessor to the root object inside the buffer.
-  let monster = Monster.getRootAsMonster(bb: ByteBuffer(bytes: buf))
+  let monster: Monster = try! getCheckedRoot(byteBuffer: &byteBuffer)
+  // let monster: Monster = getRoot(byteBuffer: ByteBuffer(bytes: buf))
```

Comment:
Is this meant to be commented out?

Reply:
Yes, we want to show users that they can use getRoot and getCheckedRoot